jupyter_sphinx/execute.py

```diff
@@ -183,6 +185,13 @@ def cell_output_to_nodes(cell, data_priority, dir):
                     text=data,
                     format='html'
                 ))
+            elif mime_type.startswith('application'):
+                if mime_type.endswith('vnd.jupyter.widget-view+json'):
+                    data = json.dumps(data)
+                to_add.append(nodes.raw(
+                    text=f'<script type="{mime_type}">{data}</script>',
```

Comment:
Embedding in a script tag doesn't seem the right thing to do for most casts of `mime_type.startswith('application')`, see [this list](https://en.wikipedia.org/wiki/Media_type#Common_examples) for examples.